### PR TITLE
Use replace_wl_with_unicode and replace_unicode_with_wl from mathics

### PR DESCRIPTION
**Pull request by GarkGarcia**
This is a follow up to https://github.com/mathics/Mathics/pull/1077.

### CODE REVIEW

File: mathicsscript/termshell.py [hunk 1]
```diff
@@ -278,7 +248,7 @@ def read_line(self, prompt):
             line = input(prompt)
         if line.startswith("!") and self.lineno == 0:
             raise ShellEscapeException(line)
-        return replace_unicode_to_wl(line)
+        return replace_unicode_with_wl(line)
```
Comment (rocky) [inline mathicsscript/termshell.py:251]: Where is `replace_unicode_with_wl` imported in this module?

**Review by rocky — COMMENTED** [review]
Note that we wil have the issue of updating the inputrc's for things like <esc>a` -> AAcuteA

File: mathicsscript/termshell.py (hunk 1 above)
Comment (GarkGarcia) [reply mathicsscript/termshell.py:252]: I forgot it 😁️

**Comment by GarkGarcia** [comment]
> Note that we wil have the issue of updating the inputrc's for things like a` -> AAcuteA

I'll try to fix the "ESC sequece alias" column in [`named-characters-data.csv`](https://github.com/Mathics3/mathics-development-guide/blob/master/resources/named-characters-data.csv) and generate the inpurc files from there.

**Comment by rocky** [comment]
> > Note that we wil have the issue of updating the inputrc's for things like a` -> AAcuteA
> 
> I'll try to fix the "ESC sequece alias" column in [`named-characters-data.csv`](https://github.com/Mathics3/mathics-development-guide/blob/master/resources/named-characters-data.csv) and generate the inpurc files from there.

Ok. Thanks for the import and working on this. If you want to merge this as is go ahead. If you want to instead fill out the importrc's and then merge that is fine too.

**Comment by GarkGarcia** [comment]
> > Note that we wil have the issue of updating the inputrc's for things like a` -> AAcuteA
> 
> I'll try to fix the "ESC sequece alias" column in [`named-characters-data.csv`](https://github.com/Mathics3/mathics-development-guide/blob/master/resources/named-characters-data.csv) and generate the inpurc files from there.

Those should be fixed with https://github.com/Mathics3/mathics-development-guide/commit/0e5cd01a3aee2676618b75ee74db4e105dc048ac. I'll generate the new inputrc file before merging this.

**Comment by GarkGarcia** [comment]
@rocky Are the entries in `inputrc-unicode` tested somewhere? Can we at-least validate that the inputrc files have valid GNU readline syntax?

**Comment by GarkGarcia** [comment]
> @rocky Are the entries in `inputrc-unicode` tested somewhere? Can we at-least validate that the inputrc files have valid GNU readline syntax?

Anyway, I'd appreciate if you could take a look at the updated version `inputrc-unicode`. I suspect there are still errors in there.

**Comment by rocky** [comment]
I don't have anything in place. If you want to add something, please do.

**Review by rocky — COMMENTED** [review]
LGTM

**Pull request merged**